python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -77,6 +100,10 @@ class RandomForestClassifier:
                         Only relevant for GLOBAL_QUANTILE split_algo.
     n_streams : int (default = 4 )
                 Number of parallel streams used for forest building
+    workers : list of strings
```

Comment:
```suggestion
    workers : optional, list of strings
```

I think we should adopt the Scikit-learn way of specifying optional parameters (I did in 773)